apps/docs/src/pages/components/tree-view.mdx

```diff
@@ -206,6 +203,7 @@ Sometimes the full tree data is unknown or paginated. This sample showcases how
 
 ```tsx live
 import { forwardRef, useState } from "react";
+import { useHvTreeItem } from "@hitachivantara/uikit-react-core";
```

Comment:
Why is this necessary now? It wasn't it case previously. Can we keep the previous behaviour?

Also, there are places where this (I believe it's the same problem) isn't being applied/resolved:

![image](https://github.com/user-attachments/assets/1bcdb8de-7829-4389-b73c-5112dd3fa340)